frontend/src/utils/config.js

```diff
@@ -0,0 +1,28 @@
+// Config values for the entire application. Ensure to document each constant!
+// See README.md for more details.
+
+//TreeViewer.jsx config
+export const VConf = Object.freeze({
```

Comment:
Can we find a better name for this? Like vConst since they're supposed to be constants?

Reply:
VConf stands for viewerConf, maybe I should make it tvconf? so it is more clear that this config refers to tree viewer?

Reply:
Okay that's fine then. tvconf is definitely more confusing so just leave it haha